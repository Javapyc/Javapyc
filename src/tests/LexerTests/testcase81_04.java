// Normal
public class Test4 {
	public static void main(String[] args) {
		while(i>0) {
			i--;
		}
	}
}