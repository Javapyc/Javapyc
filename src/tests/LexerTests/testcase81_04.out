ReservedWord, public
ReservedWord, class
ID, Test4
Delimiter, {
ReservedWord, public
ReservedWord, static
ReservedWord, void
ReservedWord, main
Delimiter, (
ReservedWord, String
Delimiter, [
Delimiter, ]
ID, args
Delimiter, )
Delimiter, {
ReservedWord, while
Delimiter, (
ID, i
Operator, >
Integer, 0
Delimiter, )
Delimiter, {
ID, i
Operator, -
Operator, -
Delimiter, ;
Delimiter, }
Delimiter, }
Delimiter, }
